### ISSUE TITLE
Source papers

### ISSUE THREAD

**Issue by Marlowess** [franec94/My-Siren-Deep-Learning-Test#1]
Do you have some links to papers describing such SIREN networks?
I'm interested in this topic.

**Comment by franec94** [comment]
Yes, I will provide you all the necessary things, so that yo can start your personal discovery about that precise topic.

However, before sending or providing you all that stuff, I want suggest you the goal of Siren Based Neural Net Architectures, that is, trying as mach as possible to identify a implicit representation - much similar to encoder-decoder paradigm in neural network , in fact I also suggest you to read the [chapter n. 14 from Ian Godfellow](https://www.deeplearningbook.org/contents/autoencoders.html) to get a grasp about encoder-decoder archs - that might be suitable to compress data informatio.

In other words, they are suitable for Deep Learning tasks related to Supervised Learning for solving regression problems, so MSE, PSNR and MSSI, are amongst the main Performance Metrics to be exploited for comparing purposes, to state some of them.

Finally, as said before, I will provide you some interesting resource, even if you mai encouter some of them earlier, that are:

- [Siren - Project Page](https://vsitzmann.github.io/siren/)
- [Siren Github](https://github.com/vsitzmann/siren)
- [Siren Paper](https://arxiv.org/abs/2006.09661)

- Usefull Python Third Party Libraries:
   - [scikit-image](https://scikit-image.org/) : exploit it mainly for looking at python's submodule metrics, whitin version 0.17.2 or upper such as 0.18.*. This library looks much similar to that of sklearn, in the sense that it provides examples and papers as do sklearn for sklearn examples. You must check for it, since it is really reach of interesting things. As an instance: [Metrics](https://scikit-image.org/docs/dev/api/skimage.metrics.html#rc4bf0a102856-1), this reference is the core reference for metrics description so you can understand all of them, even if tensorflow natively support them, so if you switch to keras+tensorflow you do right, but if you stay tuned on pytorch, I think scikit-image is the best library.
  - [tqdm](https://tqdm.github.io/): I use it for keeping track of progressing for cycles, if you look at its examples you get the point, and after you cannot stop using it.

**Comment by Marlowess** [comment]
Thanks, I appreciate it.
